app/views/posts/index.html.erb

```diff
@@ -1,9 +1,11 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>User Posts</title>
-  </head>
-  <body>
-    <h1>Posts for a given User</h1>
-  </body>
-</html>
+<div>
+  <%= render partial: 'shared/user', locals: {user: @user} %>
+
+  <% @posts.each do |post| %>
+    <%= render partial: 'shared/post', locals: {post: post} %>
+    <%= render partial: 'shared/comment', locals: {post: post} %>
```

Comment:
- [ ] Here as well, please use the method that you created in your post model which returns last 5 comments.